content/posts/jellyfin-10-5-0.md

```diff
@@ -0,0 +1,170 @@
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+**Support AMD AMF hardware encoding** is now available on Windows and Linux, as well as support for DVDs as folders, a provider for season images for TheMovieDB and various fixes to media scanning and the way base URLs and DLNA are handled. **Raspberry Pi hardware acceleration** for older models is also now supported. Sorry Raspberry Pi 4 owners, this one needs a bit more work to be supported properly.
+
+As part of a project to move the core Metadate providers to plugins, MusicBrainz now allows you to configure the URL of the instance you want to pull data from, thereby allowing you to host an instance of MusicBrainz and sidestep the global rate limiting enforced by the main service.
+
+Aside from all these improvements, more tests have been added on the server-side to help developers track down issues with new and existing code. This ensures a faster development time and prevents some breakages, which will be caught early through automated testing.
+
+## Documentation
+
+Our documentation team has also been hard at work improving both the user and developer documentation.
+
+Of note recently are the overhauls to the [Networking](https://jellyfin.org/docs/general/networking/index.html) and [Hardware Acceleration](https://jellyfin.org/docs/general/administration/hardware-acceleration.html) sections.
+
+We now provide an extensive [Codec Support](https://jellyfin.org/docs/general/clients/codec-support.html) list and help on [CSS Customization](https://jellyfin.org/docs/general/clients/css-customization.html), with examples of useful CSS customizations to apply to your server via the Administration dashboard.
+
+## Looking towards the future
+
+A lot is happening in the world of Jellyfin recently. Since the release of Jellyfin 10.4.0, we have seen a lot of new contributors joining the team and the speed of development has tremendously accelerated.
+
+With that influx of contributors, we have a few large scale projects for both the server and the web client that should, in due time, bring a ton of improvements to Jellyfin. More information is already available on [GitHub]() if you want to take part in the discussion or implementation.
+
+We'd like to thank all the contributors who worked on this release for their hard work and dedication to making the best FOSS media server possible.
+
+* [Abbe98]
+* [anthonylavado]
+* [Artiume]
+* [Bond-009]()
+* [bugfixin]()
+* [cvium]()
+* [dhartung]()
+* [dinki]()
+* [dkanada]()
+* [dmitrylyzo]()
+* [DrPandemic]()
+* [EraYaN]()
+* [ferferga]()
+* [grafixeyehero]()
+* [joshuaboniface]()
+* [JustAMan]()
+* [LogicalPhallacy]()
+* [mark-monteiro]()
+* [MrTimscampi]()
+* [Narfinger]()
+* [Nickbert7]()
+* [nvllsvm]()
+* [nyanmisaka]()
+* [oddstr13]()
+* [ploughpuff]()
+* [redSpoutnik]()
+* [sparky8251]()
+* [thornbill]()
+* [vitorsemeano]()
+* [Wunax]()
+* [YouKnowBlom]()
```

Comment:
Deadlinks galore D:

Reply:
I'm filling them up :p

Just ST: Picard is taking a bit too much of my attention.